Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.ui

```diff
@@ -82,6 +82,47 @@
          <property name="bottomMargin">
           <number>5</number>
          </property>
+         <item>
+          <widget class="AzToolsFramework::AssetBrowser::SearchWidget" name="m_searchWidget" native="true">
+           <property name="sizePolicy">
+            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
+             <horstretch>0</horstretch>
+             <verstretch>0</verstretch>
+            </sizepolicy>
+           </property>
+           <property name="minimumSize">
+            <size>
+             <width>150</width>
+             <height>0</height>
+            </size>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="Line" name="line">
+           <property name="orientation">
+            <enum>Qt::Vertical</enum>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="QToolButton" name="m_createButton">
+           <property name="text">
+            <string>...</string>
+           </property>
+           <property name="icon">
+            <iconset>
+             <normaloff>:/stylesheet/img/UI20/plus.svg</normaloff>:/stylesheet/img/UI20/plus.svg</iconset>
```

Comment:
What does this look like compared to our `img/UI20/Add.svg`. And if it is different, it seems like we should have UX investigate having a consistent plus sign image. @rainbj @amzn-leenguy

Reply:
It is the same. Deleted the Plus and replaced with Add.